tests/acceptance/customCommands/setValueBySingleKeys.js

```diff
@@ -1,9 +1,4 @@
 /**
```

Comment:
This file can be deleted as there is no need to set value by single keys now.

Reply:
it is needed for running federation tests even in chrome browser

Reply:
then please write that in the comment

Reply:
fixed

Reply:
In that case please fix this function while you're at it. For the federated sharing problem we don't need to send value by single keys. We just need to wait a short time while entering last few characters.
change line 15 to
``` js
 const charPromise = this.setValue(selector, chars)
```
change line 21 to
``` js
 return Promise.all([charPromise, ...charEndPromise])
```
That should fix the problem IMO but may need some tweaking